contrib/automation_tests/README.md

```diff
@@ -108,4 +108,9 @@ Running an E2E test with the `-dev_mode` command line parameter will:
 - Not close Orbit when a test is done
 
 Use this for quick iterations on new tests. In addition, omit the usual test cases to connect to the 
-stadia instance and select a process while you're developing, so you can simply keep Orbit open.  
\ No newline at end of file
+stadia instance and select a process while you're developing, so you can simply keep Orbit open.
+
+### Style
+
+[yapf](https://github.com/google/yapf) is used with the Google style to keep consistent code style
```

Comment:
Can you add a one line tldr; how to invoke yapf? 

I'll 'pip install' it to the zip file containing the python installation so it is available right away.

Reply:
I added more information, including a line how to invoke it.

> I'll 'pip install' it to the zip file containing the python installation so it is available right away.
thanks for that!